setup.py

```diff
@@ -11,7 +11,7 @@
 
 setup(
     name="revChatGPT",
-    version="4.0.12",
+    version="4.1.0",
```

Comment:
Please don't bump my versions for me

Reply:
Oh sorry for that, I thought it will be convenient for you🫠.